src/game/gameflow.c

```diff
@@ -254,12 +254,18 @@ static bool GameFlow_LoadScriptGameStrings(struct json_object_s *obj)
         if (!value || !value->string || key < 0 || key >= GS_NUMBER_OF) {
             LOG_ERROR("invalid string key %s", strings_elem->name->string);
         } else {
-            Memory_FreePointer(&g_GameFlow.strings[key]);
```

Comment:
This is handled by calling `GameFlow_Shutdown()` before re-initialization of the gameflow.